static/css/components/dropper.less

```diff
@@ -68,8 +68,50 @@
     }
   }
 }
-
 /* stylelint-disable selector-max-specificity */
+
+.my-reading-lists {   
```

Comment:
> The javascript fails because of increase in size of page-book.css which exceeds its maximum value.

This is flagging something quite important - that we are increasing the amount of CSS we ship to the page, and likely slowing down the page load.

Here it doesn't seem like we want/need to do that.

If we're moving something, there should be no impact on the change of CSS - existing CSS should either get replaced or revised.

If replacing, which is what I think you are doing here, this should be a new component and the dropper.less component should no longer be added to the page.

Reply:
Properties of "reading-lists" doesn't apply to this class even when I haven't renamed the className to "my-reading-lists". This is because the class "reading-lists" is defined under "dropdown" class in "less" and I am bringing this part out of this "dropdown" class. Hence I renamed the className to "my-reading-lists" and copied the properties into this class in dropper.less. But, the properties I have added under my-reading-lists class in dropper.less is not reflected in the site.

Reply:
Are you repurposing the dropper-list component [1] or creating a new component?


[1] https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#dropper-list

Reply:
I was creating a new component with className "my-reading-lists" specifically for My Reading Lists part. But I deleted this part of code because it was not working. It can be added if we want to format that part.

Reply:
If you are creating a new component, then because of your changes to openlibrary/templates/lists/widget.html  the component "dropper" no longer exists as it is no longer used. I would therefore suggest amending the existing styles in this stylesheet and possibly renaming the component, rather than appending new styles and changing the HTML styling class.

Reply:
"dropdown" is used but "My Reading Lists" is no loner part of "dropper" div.

Reply:
Then we should pull out that CSS and associated styles into a new component calling reading-list.less